Makefile

```diff
@@ -71,10 +73,10 @@ kops-cluster-delete:
 	kops delete --state=${KOPS_STATE_STORE} -f hack/kops-aws-usw2.cluster.yaml --yes
 
 kind-cluster-config:
-	export KUBECONFIG=$$(kind get kubeconfig-path --name="kind")
+	export KUBECONFIG=$$(kind export kubeconfig --name="kind")
```

Comment:
Should get rid of kind cluster creation from the Makefile.
Its test dependencies.

Reply:
We have this to help test `make bdd`, I tested that works although bdd's are failed for 1 case.